myscripts.js

```diff
@@ -1,81 +1,169 @@
+ }
+
+ booksHtmlTemplate = (obj) => `<div class='bookCont'>
+ <label class='title'>${obj.title}</label><br>
+ <label id='author'>${obj.author}</label><br>
+ <button class='remove' type='button' onclick= value=''>Remove</button>
+ <hr>
+ </div>`;
 }
 
-// FUNCTON DECLARATIONS
+let bookmgr = new BookManager();
+// SELECTORS
 
-const addbook = function () {
-  const titleinput = document.getElementById('titleInput').value;
-  const authorinput = document.getElementById('authorInput').value;
-  storageObject = JSON.parse(localStorage.getItem('books'));
-  if (titleinput !== '' && authorinput !== '') {
-    const book = new Book(titleinput, authorinput);
-    if (storageObject === '') {
-      storageObject.push(book);
-      localStorage.setItem('books', JSON.stringify(storageObject));
-      bookContainer.innerHTML += booksHtmlTemplate(book);
-    } else {
-      collection.push(book);
-      localStorage.setItem('books', JSON.stringify(collection));
-      bookContainer.innerHTML += booksHtmlTemplate(book);
-    }
-    update();
-  }
-};
+// const addbtn = document.getElementById('add');
+// title = document.getElementById('title');
+// const bookContainer = document.getElementById('bookContainer');
+// new let storageObject = '';
 
-// EVENT LISTENERS
+// new const booksHtmlTemplate = (obj) => `<div class='bookCont'>
+//   <label class='title'>${obj.title}</label><br>
+//   <label id='author'>${obj.author}</label><br>
+//   <button class='remove' type='button' onclick= value=''>Remove</button>
+//   <hr>
+//   </div>`;
 
-addbtn.addEventListener('click', addbook);
\ No newline at end of file
+// new const createBook = (arr) => {
+//   arr.forEach((element) => {
+//     const dinContent = document.createDocumentFragment();
+//     const newDiv = document.createElement('div');
+//     newDiv.innerHTML = DocAccesor.booksHtmlTemplate(element);
+//     dinContent.appendChild(newDiv);
+//     DocAccesor.bookContainer.appendChild(dinContent);
+//   });
+// };
+
+// new2 const update = () => {
+//   const elements = document.getElementsByClassName('remove');
+//   const bookCont = document.querySelectorAll('.bookCont');
+//    bookmgr.collection = JSON.parse(localStorage.getItem('books'));
+//   Array.from(elements).forEach((btn, index) => {
+//     btn.addEventListener('click', () => {
+//       if (bookCont[index].parentNode) {
+//         bookCont[index].parentNode.removeChild(bookCont[index]);
+//         bookmgr.collection.splice(index, 1);
+//         localStorage.setItem('books', JSON.stringify(bookmgr.collection));
+//       }
+//     });
+//   });
+// };
+
+// OBJECT CONSTRUCTOR FUNCTION
+
+// new3 window.addEventListener('load', () => {
+//   bookmgr.storageObject = JSON.parse(localStorage.getItem('books'));
+//   if (bookmgr.storageObject) {
+//     bookmgr.createBook(bookmgr.storageObject);
+//     bookmgr.update();
+//   } else {
+//     DocAccesor.bookContainer.innerHTML = '';
+//   }
+// });
+
+// FUNCTON DECLARATIONS
+
+// EVENT LISTENERS
```

Comment:
[Optional] Can we remove all this commented code? 🤔, it makes the code look unprofessional 🙂 

Kindly check for other occurrences 👍